plugins/mcreator-localization/help/default/block/block_states.md

```diff
@@ -0,0 +1,5 @@
+Here you can list additional custom properties of this block.
+
+Some properties are used internally (for block rotation and waterlogging), so their names can not be used.
+
+Along with custom ones, you can also use pre-defined block state properties used by vanilla Minecraft blocks.
```

Comment:
This is not true for the state of this UI, so this help tip should not show this yet